src/input_handling.rs

```diff
@@ -79,13 +80,13 @@ fn substitute_command_parameters(
 /// wouldn't be able to construct a command from them if they cannot be converted to
 /// valid UTF-8.
 pub fn execute_triggered_commands(
-    config: Rc<RefCell<Configuration>>,
+    config: Arc<Mutex<Configuration>>,
```

Comment:
For my own understanding, why does this change to `Arc<Mutex<Configuration>>`?  I'm guessing this is accessed from another thread?

Reply:
Honestly, I think this code is way over-engineered.  It might be needed because it's called from glutin, idk?  The old "cache" file used Arc<Mutex<Cache>> in this way, so when I moved that functionality to Configuration, I updated it.

Reply:
Yeah, it is - this can be updated later.